app/views/layouts/application.html.erb

```diff
@@ -48,7 +48,7 @@
     <%= javascript_include_tag 'vue_navigation_top_menu' %>
   </head>
   <body
-    class="<%= yield :body_class %> <%= 'navigator-collapsed' if !@navigator || session[:navigator_collapsed] %>"
+    class="<%= yield :body_class %> <%= 'navigator-collapsed' if !@navigator || session[:navigator_collapsed] %> <%= 'w-98' if params[:controller] == 'label_templates' && params[:action] == 'show'%>"
```

Comment:
You can use `w-[98%]` if you need a specific width:
https://tailwindcss.com/docs/width#arbitrary-values